src/common/libterminus/client.c

```diff
@@ -316,6 +312,10 @@ static void pty_die (struct flux_pty_client *c, int code, const char *message)
         free (c->exit_message);
         c->exit_message = strdup (message);
     }
+    if (c->attached && (c->flags & FLUX_PTY_CLIENT_NOTIFY_ON_DETACH)) {
+        printf ("\033[999H[detached: %s]\033[K\n\r", c->exit_message);
+        fflush (stdout);
+    }
```

Comment:
not knowing the code super well, now that `message` can be NULL, is there a scenario where `exit_message` has not yet been set?  Perhaps a small comment to indicate it will have been set by this point is all that's important.

Reply:
Good point. Maybe it would be better to handle the case where the `exit_message` is NULL by printing `unknown reason`, though it is likely that this case can't be covered in testing.

Reply:
Ok, I think I addressed your comment in the latest push.